src/app/page.tsx

```diff
@@ -1,116 +1,132 @@
+							streaming. Our platform utilizes advanced encryption and
+							decentralized technology to ensure your privacy and freedom of
+							expression. Discover a wide range of diverse and uncensored
+							content from creators worldwide. Join us today and experience true
+							freedom in streaming.
+						</i>
+					</p>
+				</div>
+				<div className="h-2/5 px-28 bg-[url('/assets/wave.svg')] bg-cover mt-5 w-full">
+					<button className="w-32 h-8 mx-4 bg-blue-950 text-white hover:border-2 hover:border-blue-950 hover:bg-white hover:text-black ">
+						Try
+					</button>
+					<button className="w-32 h-8 mx-4 border-2 border-blue-950 hover:bg-blue-950 hover:text-white">
+						Learn more
+					</button>
+				</div>
+			</div>
+			<div className="flex flex-col items-center w-full mt-20">
+				<div className="w-11/12 md:w-9/12 lg:w-6/12 flex items-center my-4">
+					<div className="w-5/6">
+						<h3 className="text-2xl font-bold mb-4">
+							A virtual stage for everything
+						</h3>
+						<p className="text-sm">
+							Step into our virtual world and experience the thrill of live
+							performances, exhibitions, workshops, and so much more, all from
+							the comfort of your own home. No longer limited by physical
+							boundaries, our virtual stage breaks down barriers and brings
+							people together from every corner of the globe.
+						</p>
+					</div>
+					<img
+						src="/assets/undraw1.svg"
+						alt=""
+						className="w-[200px] h-[200px] m-2"
+					/>
+				</div>
+				<div className="w-11/12 md:w-9/12 lg:w-6/12 flex items-center my-4">
+					<img
+						src="/assets/undraw1.svg"
+						alt=""
+						className="w-[200px] h-[200px] m-2"
+					/>
+					<div className="w-5/6">
+						<h3 className="text-2xl font-bold mb-4">
+							Encrypted chat on-demand
+						</h3>
+						<p className="text-sm">
+							SEnd-to-end encryption guarantees that only you and the intended
+							recipients can access the messages exchanged. This means that not
+							even our platform administrators or any third parties can decipher
+							or intercept your communications. Your data remains confidential
+							and inaccessible to anyone else.
+						</p>
+					</div>
+				</div>
+				<div className="w-11/12 md:w-9/12 lg:w-6/12 flex items-center my-4">
+					<div className="w-5/6">
+						<h3 className="text-2xl font-bold mb-4">
+							Follow the creators you love
+						</h3>
+						<p className="text-sm">
+							Our platform is designed to foster a symbiotic relationship
+							between creators and their fans, cultivating a vibrant ecosystem
+							where creativity thrives. Together, we celebrate the power of art
+							and honor the journeys of those who bring it to life.
+						</p>
+					</div>
+					<img
+						src="/assets/undraw1.svg"
+						alt=""
+						className="w-[200px] h-[200px] m-2"
+					/>
+				</div>
+				<div className="w-11/12 md:w-9/12 lg:w-6/12 flex items-center my-4">
+					<img
```

Comment:
Why don't you use Next/Image?

Reply:
Apparently, there is a slight problem with next.js(the app dir version) Image component. They improved it but apparently, that introduced a performance bug for a while that's why I kind of ignored it for the moment. Do you think I should still put it ?

Reply:
I think it's fine, just wanted to get the reason